diff --git a/api/PrepaidCardController.php b/api/PrepaidCardController.php
index b80772e..51d7748 100644
--- a/api/PrepaidCardController.php
+++ b/api/PrepaidCardController.php
@@ -1,5 +1,5 @@
 <?php
-class TestController
+class PrepaidCardController
 {
     /**
      * Returns a JSON string object to the browser when hitting the root of the domain
diff --git a/api/index.php b/api/index.php
index 2361f55..26e4d40 100644
--- a/api/index.php
+++ b/api/index.php
@@ -1,7 +1,7 @@
 <?php

 require __DIR__ . '/vendor/jacwright/restserver/source/Jacwright/RestServer/RestServer.php';
-require 'TestController.php';
+require 'PrepaidCardController.php';

 $server = new \Jacwright\RestServer\RestServer('debug');
 $server->useCors = true;
